lib/avro_turf/connection_wrapper.rb

```diff
@@ -37,8 +37,8 @@ def initialize(
     @client_cert_data = client_cert_data
     @client_key_data = client_key_data
     @connection_pool_size = connection_pool_size || CONNECTION_POOL_SIZE
-    @tcp_nodelay = tcp_nodelay || TCP_NODELAY
-    @persistent_connection = persistent_connection || PERSISTENT_CONNECTION
+    @tcp_nodelay = tcp_nodelay.nil? ? TCP_NODELAY : tcp_nodelay
```

Comment:
not related, but any reason for this? the former is a pretty common rails pattern. (was there something else falsey that we might have been passing in here?)

Reply:
if tcp_nodelay was set to false we were setting it to true anyway

Reply:
false || true

Reply:
got it, so there were falsey things being passed in. :+1:

Reply:
not in our case but potentially yes